src/pages/recurring-invoices/common/hooks/useResolveInputField.tsx

```diff
@@ -31,11 +37,37 @@ interface Props {
 
 export function useResolveInputField(props: Props) {
   const handleProductChange = useHandleProductChange();
+  const [currency, setCurrency] = useState<Currency>();
+
   const onChange = useHandleLineItemPropertyChange();
   const [t] = useTranslation();
   const { setIsTaxModalOpen, setIsProductModalOpen } = props;
+
   const invoice = useCurrentRecurringInvoice();
   const formatMoney = useFormatMoney();
+  const company = useCurrentCompany();
+
+  const clientResolver = new ClientResolver();
+  const currencyresolver = new CurrencyResolver();
+
+  const getCurrency = async (client_id: string) => {
```

Comment:
Is it possible to extract getCurrency method into custom hook so we can reuse it in both components?